docs/standard/memory-and-spans/memory-t-usage-guidelines.md

```diff
@@ -114,7 +114,8 @@ Using a parameter of type <xref:System.Span%601> instead of type <xref:System.Me
 
 Sometimes, you'll have to use a <xref:System.Memory%601> parameter instead of a <xref:System.Span%601> parameter, even if you're fully synchronous. Perhaps an API that you depend accepts only <xref:System.Memory%601> arguments. This is fine, but be aware of the tradeoffs involved when using <xref:System.Memory%601> synchronously.
 
-<a name="rule-2 />
+<a name="rule-2></a>
```

Comment:
The problem here, I think, wasn't that `</a>` needs to be a separate element; it's that `rule-2` does not have a closing quotation mark.

Reply:
Duh, I missed that.